### PR TITLE
Fix full repay borrow

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/25drfkk

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2201k46 Design QA](https://app.clickup.com/t/2201k46)

**Comment by benluelo** [comment]
This has already been implemented: https://github.com/ComposableFi/composable/blob/ben/lending-tests-wip/frame/lending/src/lib.rs#L2029-L2036

File: frame/lending/src/tests.rs [hunk 1]
```diff
@@ -673,7 +679,8 @@ fn test_repay_partial_amount() {
 		dbg!(alice_balance);
 		dbg!(Tokens::balance(COLLATERAL::ID, &ALICE));
 		dbg!(Tokens::balance(COLLATERAL::ID, &BOB));
-		assert!(alice_balance > Tokens::balance(COLLATERAL::ID, &ALICE));
+		// TODO: fix bug with partial repay
```
Comment (dzmitry-lahoda) [inline frame/lending/src/tests.rs:682]: no sure, the test is that repay all at once == repay several times in part within same block, given accrued interest is not equal to zero

**Comment by benluelo** [comment]
Closed as #867 has the same fix.

**Pull request closed without merging**